.gitattributes

```diff
@@ -0,0 +1,67 @@
+# Set default behavior to automatically normalize line endings.
+* text=auto
+
+*.doc  diff=astextplain
+*.DOC	diff=astextplain
```

Comment:
Shouldn't .doc be binary?

Reply:
Everywhere I look, these are all `astextplain`.  Consensus isn't correctness, but I have to dig deeper into why that's the way it is.

Reply:
Interesting, so it seems there is a common [shell script](https://github.com/msysgit/msysgit/blob/master/bin/astextplain) that is used to do conversions into text from word and PDF docs.  The funny thing is, dotnet/runtime doesn't have this script at all and no one mentioned `msysgit`.  

In any case, I don't see a real reason to not mark these as binary for our purposes, I'll change them up.